components/QueryToken/QueryToken.tsx

```diff
@@ -0,0 +1,21 @@
+﻿import React from "react";
```

Comment:
Could we rename to something that keeps the word `Link` in the name?
Perhaps `TokenLink` or something?

Reply:
changed name for clarity